d/build.gradle

```diff
@@ -1,37 +1,20 @@
 plugins {
-    id 'scala'
```

Comment:
Yeah, noticed that your plugin applies the Scala plugin internally by looking at the code. Is that documented somewhere? Didn't see that.

Reply:
It is not mention explicitly, But the examples supports the exclusion :) I do see a point in mentioning it explicitly though